cypress/support/PageObject.js

```diff
@@ -1,7 +1,159 @@
+/// <reference types="cypress" />
+
 class PageObject {
   visit(url) {
     cy.visit(url || this.url);
   }
+
+  get usernameLink() {
+    return cy.getByDataQa('link-username');
+  }
+
+  get pageTitle() {
+    return cy.getByDataQa('title-page');
+  }
+
+  get articleBlock() {
```

Comment:
Create separate page objects for each separate page. In PageObject.js we should have only main common methods

Reply:
But I thought that **PageObject** could contain different common methods, not only main. I thought that inside **PageObject** we could keep common methods for the entire app to avoid the redundant availability of the identical methods across different pages. So, it serves as the basement.

Reply:
But if I'm not right on that point, correct me, please, and I'll fix it immediatelly :)